src/common/hotbar-store.ts

```diff
@@ -44,8 +44,8 @@ export class HotbarStore extends BaseStore<HotbarStoreModel> {
       },
       migrations,
     });
+
     makeObservable(this);
-    this.load();
```

Comment:
Why was this removed? I cannot find where you guys readded this. So this is certainly a bug

Reply:
As you noticed, it's moved to controlled place. This is because of the temporal dependencies between syncing general entities and hotbar store.